indigo_api/views/attachments.py

```diff
@@ -49,6 +69,8 @@ def download(self, request, *args, **kwargs):
     @action(detail=True, methods=['GET'])
     def view(self, request, *args, **kwargs):
         attachment = self.get_object()
+        if attachment.mime_type in DOC_MIMETYPES:
+            return view_attachment_as_pdf(attachment)
```

Comment:
@longhotsummer Similar to the reasoning in the js, I think this logic describes our goals a bit better i.e  we are interested in converting docx types only rather that putting everything through soffice.